test/ci/try_assert.cc

```diff
@@ -2,6 +2,10 @@
 #include <iostream>
 
 int main(int, char**) {
+#if defined(_MSC_VER)
+  // Do not display message box on abort.
+  _set_abort_behavior(0, _WRITE_ABORT_MSG | _CALL_REPORTFAULT);
```

Comment:
Flags like this need documentation about what they do.

Had this been done for `_CALL_REPORTFAULT`, which "invokes the Windows Error Reporting Service mechanism [...] to report failures to Microsoft", it would be rather more obvious that we don't want this flag set.

Reply:
Both these flags are cleared. The second parameter is the mask to specify which flags to change, and the first flag is zero, to clear both of them.

I updated the comment.